packages/ui/src/containers/SocialSignIn/PrimarySocialSignIn.tsx

```diff
@@ -19,15 +19,15 @@ const PrimarySocialSignIn = ({ className, isPopup = false, onSocialSignInCallbac
   const [showAll, setShowAll] = useState(false);
   const { invokeSocialSignIn, socialConnectors } = useSocial({ onSocialSignInCallback });
   const isOverSize = socialConnectors.length > defaultSize;
-  const displayAll = showAll || isPopup || !isOverSize;
+  const displayAll = isPopup || !isOverSize;
 
   const displayConnectors = useMemo(() => {
-    if (displayAll) {
+    if (displayAll || showAll) {
```

Comment:
nit: `displayAll` and `showAll` are a little bit confusing. I don't know, maybe we should rename the former to `shouldDisplayAll`?

Reply:
make sense.  updated